src/Microsoft.Azure.SignalR.AspNet/ServerConnections/ServiceConnection.Log.cs

```diff
@@ -11,6 +11,9 @@ internal partial class ServiceConnection
         private static class Log
         {
             // Category: ServiceConnection
+            private static readonly Action<ILogger, int, Exception> _startToCleanupClientConnections =
+                LoggerMessage.Define<int>(LogLevel.Information, new EventId(1, "StartToCleanupClientConnections"), "Start to cleanup {clientCount} client connections, can be triggered by a service connection drop or app server shutdown.");
```

Comment:
`Closing {clientCount} client connections for server connection {serverConnectionId} (and instanceId {fromInstanceId} when it is not null).`